main.py

```diff
@@ -4,7 +4,7 @@
 from omegaconf import DictConfig, OmegaConf
 import wandb
 
-from ufm import countermeasures, fine_tuning, models, pretrain_score, unadapt, utils
+from src.ufm import countermeasures, models, unadapt, utils, metrics, fine_tuning
```

Comment:
You shouldn't use src.ufm if you have the latest changes and did `pip install -e .`